prow/cmd/deck/static/prow/prow.ts

```diff
@@ -393,17 +393,23 @@ function redraw(fz: FuzzySearch): void {
         return sel;
     }
 
-    function getSelectionFuzzySearch(id: string, inputId: string): string {
+    function getSelectionFuzzySearch(id: string, inputId: string): RegExp {
         const input = document.getElementById(inputId) as HTMLInputElement;
         const inputText = input.value;
-        if (inputText !== "" && opts && !opts[id + 's' as keyof RepoOptions][inputText]) {
-            return "";
+        if (inputText === "") {
+            return new RegExp('');
+        }
+        if (inputText !== "" && opts && opts[id + 's' as keyof RepoOptions][inputText]) {
+            return new RegExp(`^${inputText.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')}$`);
```

Comment:
I think it is sufficiently non-obvious what `inputText.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')` does that it might be worth pulling it out into an `escapeRegex` helper function or something (especially given it's used more than once).

Reply:
Yeah, will do.